framework/CodeInterfaces/Dymola/DymolaInterface.py

```diff
@@ -232,7 +232,7 @@ def createNewInput(self, currentInputFiles, oriInputFiles, samplerType, **Kwargs
           inc = 0
           Fvect.write("double %s(%s,2) #Comments here\n" %(key, len(value)))
           for val in value:
-            Fvect.write("%s\t%s\n" %(inc,val[0]))
+            Fvect.write("%s\t%s\n" %(inc,val))
```

Comment:
This variable name might technically be in violation of the code standards, but it appears to be grandfathered in, so no change necessary.